### ISSUE TITLE
No module named 'pythagorean' when import numerology

### ISSUE THREAD

**Issue by yerzhan-karatay** [compuuteio/numerology#4]
Hi @eguenou , I'm facing an error when testing numerology in my local. it seems the numerology init file can not access `pythagorean` package.

Versions:
```
pip 21.1.3 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)
Python 3.9.6
```

Code:
```
from numerology import PythagoreanNumerology
```

Error:
```
ModuleNotFoundError: No module named 'pythagorean'
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-4-34674351218f> in <module>
----> 1 from numerology import PythagoreanNumerology

/usr/local/lib/python3.9/site-packages/numerology/__init__.py in <module>
      4 import sys
      5 
----> 6 from pythagorean.common import Functions as fct
      7 from pythagorean.numerology import Numerology as Pythagorean
      8 

ModuleNotFoundError: No module named 'pythagorean'
```

I haven't checked the codebase of your package, but if you comment [this part](https://github.com/compuuteio/numerology/blob/master/numerology/__init__.py#L6-L7) you will see another error.

**Comment by dvwright** [comment]
I made a PR which resolves the issue.

To test the PR from the command line:
```bash
$ virtualenv tst
$ source tst/bin/activate
(tst)$ pip install git+https://github.com/dvwright/numerology.git

$ cat>test_nume.py<<EOF
from numerology import Pythagorean
his_numerology = Pythagorean("Barrack", "Obama", "1961-08-04")
EOF

$ python3 test_nume.py
```

**Comment by eguenou** [comment]
Hi @dvwright,
Thanks for your help! I merged your correction and it does work.
FYI, in the project, you can also run tests by doing : python -m unittest tests/test_numerology.py.

**Comment by eguenou** [comment]
Hi,
I did my tests with Python 3.9. The project was started with 3.7 and @dvwright changes was for 3.6.
I'll try this week to put some consistency on the Python version so it would work on every configuration.
Sorry for the delay and the disappontements.

**Comment by eguenou** [comment]
Hi,
All is fixed and tested with Python 3.8 in the version Numerology 1.8.
